### DISCUSSION TITLE
app for esp8266?

### DISCUSSION THREAD

**Discussion by mgardenal** [arendst/Tasmota#15781] [Q&A] [answered]
I'm new to tasmota, I use Blynk, they will shut down the servers.
I'm looking for a similar alternative and I met Tasmota
But I'm having trouble finding an app to use my phone to turn the lights on and off
It's possible?
I read in another thread that Ewelink no longer works? i was trying to run pairing mode on esp8266 but i couldn't keep the led blinking

**Comment by rj10rd** [comment]
Setup your tasmota using this mqtt cloud server
 https://www.hivemq.com/mqtt-cloud-broker/ 

then use this app https://play.google.com/store/apps/details?id=com.lapetov.mqtt

**Comment by pkkrusty** [comment, marked as answer]
Or if you want to keep things local, get a Raspberry Pi Zero 2 W for $15 and run a MQTT server yourself. Another option is using Hue emulation and using an Echo Dot, then you have remote access through the Alexa app.

**Comment by Jason2866** [comment]
There is no official supported Mobile App for Tasmota. Tasmota is NOT designed to be directly connected to the internet. A remote control can be realized with a Home Automation Solution and using there App to control the Tasmota devices.

**Comment by MacSass** [comment]
There is an app called „Tasmotrol“ for iOS and Android, that does everything local via http.

For Android there are other apps like „ Tasmota Control “ as well with similar functionality.
